src/sdk/PropertyId.ts

```diff
@@ -245,6 +245,13 @@ export enum PropertyId {
      */
     SpeechServiceConnection_EnableAudioLogging,
 
+    /**
+     * A string value representing the priority for continuous language detection.
+     * Allowed values are "Latency", and "" (for no continous language detection). "Accuracy" is currently unsupported for Continuous LID.
+     * Added in version 1.21.0
+     */
+    SpeechServiceConnection_ContinuousLanguageIdPriority,
```

Comment:
Why do you allow setting an empty string? (""). I think you should just say that only "Latency" is supported. 

To set at-start LID, there is another property called "SpeechServiceConnection_SingleLanguageIdPriority" which takes values "latency" or "accuracy". I have a work item to change the name to  SpeechServiceConnection_AtStartLanguageIdPriority (and mark the original property as deprecated). I don't see you have that property in JS. Can you add it? How do you currently switch between latency and accuracy in JS for LID with SR? 
The default is at-start LID with Latency mode.

Reply:
Let's have a quick phone call if you want to discuss this.